tests/test_utilities.py

```diff
@@ -679,6 +689,7 @@ def test_send_email_notification_message_with_site_from_email_override(
         utils.send_email_notification_message(
             user,
             enrolled_in,
+            dashboard,
```

Comment:
for consistently I would also change these vars to dashboard_url but otherwise lgtm!